department-of-veterans-affairs/vets-website
# Pensions: Add transferred assets

## Summary

On Step 5 of Financial Information: net worth,

- create a new page for the Did you, your spouse, or your dependents transfer any assets in the last 3 calendar years? question
- Include blue info box with expandable copy
- Add copy to provide examples of money transfers
- Add gray warning box if they answer "No"
- match all copy to designs

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the pension_form_enabled flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run vets-website and vets-api

## How to verify
1. Go to http://localhost:3001/pension/application/527EZ in your browser
2. Select 'Start your application without signing in'
3. Proceed to Chapter 5 by filling out the required fields in previous chapters
4. On the question, "Did you, your spouse, or your dependents transfer any assets in the last 3 calendar years?", confirm that the page matches the designs

## Related issue(s)

- [Link to GitHub Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/70410)
- [Link to Epic](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done


## Screenshots

| Before | After | After |
| ------- | ----------------|--------------|
|  N/A (new question) | <img width="613" alt="Screenshot 2023-12-11 at 12 43 08 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/6d9db7f3-ce19-418c-b1d8-d82139bdf5c2"> | <img width="580" alt="Screenshot 2023-12-11 at 12 43 19 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/c45e57cc-b0ed-4f16-baed-5e9eb43316e4"> |

## What areas of the site does it impact?

Pension Benefits

## Acceptance criteria

- [Link to GitHub Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/70410)

### Quality Assurance & Testing


### Error Handling


### Authentication
